build_package.py

```diff
@@ -15,12 +15,27 @@
 import glob
 import os
 import shutil
+import stat
 import subprocess
 import sys
 import zipfile
 
 from playwright.path_utils import get_file_dirname
 
+driver_version = "0.5.2"
+
+if not os.path.exists("driver"):
```

Comment:
Duplicate logic with the paths. I would use Pathlib for that.

Reply:
Those are different folders or what do you mean?